node/forest_libp2p/Cargo.toml

```diff
@@ -5,10 +5,11 @@ authors = ["ChainSafe Systems <info@chainsafe.io>"]
 edition = "2018"
 
 [dependencies]
-libp2p =  { git = "https://github.com/SigP/rust-libp2p", rev = "776d13ef046358964c7d64cda3295a3a3cb24743" }
 utils = { path = "../utils" }
-tokio = "0.1.22"
-futures = "0.1.29"
+libp2p = "0.15.0"
+futures = "0.3.1"
+futures-util = "0.3.1"
 log = "0.4.8"
 slog = "2.5.2"
+async-std = {version  = "1.4.0", features = ["unstable"]}
```

Comment:
Nit: Fix spacing `{ version  = "1.4.0", features = ["unstable"] }`

Reply:
Wish  they had a linter for cargo files 😂

Reply:
There is a TOML formatter though! I use one with VSCode, but it's not perfect

Reply:
fixed